docs/elasticsearch_plugins.rst

```diff
@@ -96,16 +96,19 @@ To make this work, you need to manually edit Rally's configuration file in ``~/.
 
     plugin.my-plugin.remote.repo.url = git@github.com:example-org/my-plugin.git
     plugin.my-plugin.src.subdir = elasticsearch-extra/my-plugin
-    plugin.my-plugin.build.task = :my-plugin:plugin:assemble
+    plugin.my-plugin.build.command = ./gradlew :my-plugin:plugin:assemble
     plugin.my-plugin.build.artifact.subdir = plugin/build/distributions
 
 Let's discuss these properties one by one:
 
 * ``plugin.my-plugin.remote.repo.url`` (optional): This is needed to let Rally checkout the source code of the plugin. If this is a private repo, credentials need to be setup properly. If the source code is already locally available you may not need to define this property. The remote's name is assumed to be "origin" and this is not configurable. Also, only git is supported as revision control system.
 * ``plugin.my-plugin.src.subdir`` (mandatory): This is the directory to which the plugin will be checked out relative to ``src.root.dir``. In order to allow to build the plugin alongside Elasticsearch, the plugin needs to reside in a subdirectory of ``elasticsearch-extra`` (see also the `Elasticsearch testing documentation <https://github.com/elastic/elasticsearch/blob/master/TESTING.asciidoc#building-with-extra-plugins>`_.
-* ``plugin.my-plugin.build.task`` (mandatory): The Gradle task to run in order to build the plugin artifact. Note that this command is run from the Elasticsearch source directory as Rally assumes that you want to build your plugin alongside Elasticsearch (otherwise, see the next section).
+* ``plugin.my-plugin.build.command`` (mandatory): The full Gradle command to run in order to build the plugin artifact. Note that this command is run from the Elasticsearch source directory as Rally assumes that you want to build your plugin alongside Elasticsearch (otherwise, see the next section).
 * ``plugin.my-plugin.build.artifact.subdir`` (mandatory): This is the subdirectory relative to ``plugin.my-plugin.src.subdir`` in which the final plugin artifact is located.
 
+.. warning::
```

Comment:
I think we can skip this warning but I think we should add a note to the migration docs? (no harm in leaving it here but I'd definitely add a note to the migration docs)

Reply:
I have seen that you also refer to this location in your warning message while migrating the config. In that I think we should keep it here as well.